src/com/amplitude/api/AmplitudeClient.java

```diff
@@ -272,6 +272,7 @@ protected long logEvent(String eventType, JSONObject eventProperties,
             event.put("country", replaceWithJSONNull(deviceInfo.getCountry()));
             event.put("language", replaceWithJSONNull(deviceInfo.getLanguage()));
             event.put("platform", Constants.PLATFORM);
+            event.put("limit_tracking_enabled", replaceWithJSONNull(deviceInfo.isLimitAdTrackingEnabled()));
```

Comment:
Also I don't think you need the replaceWithJSONNull since isLimitAdTrackingEnabled returns a boolean primitive